libraries/rainbow/CMakeLists.txt

```diff
@@ -8,15 +8,7 @@ set(RAINBOW_SOURCES
 src/rainbow.cpp
 )
 
-add_definitions(-DRAINBOW_LIBRARY)
```

Comment:
Maybe revert 61db1c46beb465c33124ec4f34dfdcefd4d804d3 first and then apply your changes. That way all libraries should be called `Launcher_`. 61db1c46beb465c33124ec4f34dfdcefd4d804d3 was just a workaround to allow installing PolyMC and MMC side-by-side